edgedb-repl/src/commands/configure.rs

```diff
@@ -37,7 +37,8 @@ pub async fn configure(cli: &mut Client<'_>, _options: &Options,
                 props.push(format!("user := {{ {} }}", users))
             }
             if comment.is_some() {
-                props.push(format!("comment := {:?}", comment))
+                props.push(format!(
+                    "comment := {}", quote_string(comment.as_ref().unwrap())))
```

Comment:
``` 
if let Some(ref comment_text) = comment {
   ... quote_string(comment_text)... 
}
```
This avoids `unwrap` and is generally cleaner.

Reply:
Nice. Thanks for the tip.